src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1 class="title">Leaderboard</h1>
+    <div class="d-flex">
+      <section class="leaderboard-show">
+        <div class="leaderboard-title d-flex justify-content-between">
+          <h2 class="fw-bold fs-1">Recent scores</h2>
+          <button class='button' id='refreshBtn'>Refresh</button>
+        </div>
+        <div class="scores-div">
+          <ul class="leaderboard-scores" id='leaderboardScores'></ul>
+        </div>
+      </section>
+      <section class="leaderboard-form" id='form'>
+        <div class="leaderboard-title d-flex justify-content-between">
+          <h2 class="fw-bold fs-1">Add your score</h2>
+        </div>
+        <label for="userName">Your name</label>
+        <input class='text-center' type="text" name="username" id="userName" placeholder="e.g: John Wick">
+        <label class='mt-3' for="userScore">Your score</label>
+        <input class='text-center' type="number" name="userscore" id="userScore" placeholder="e.g: 209">
+        <div class="d-flex justify-content-between align-items-center">
+          <p class="d-none error m-0" id='error'>Invalid Input!</p>
+          <input class='mt-3 button submit' type="submit" name="submitBtn" id="submitBtn" value='Submit'>
+        </div>
+      </section>
+    </div>
```

Comment:
- [ ] As per the requirement in `index.html` you just have to create a root element and all the code will be added from `javascript`, it will make your page more dynamic and also make sense of using `javascript`. :slightly_smiling_face:

Reply:
I'm sorry, I don't understand this part. Just the scoreboard is dynamic, no? Do I need to do everything dynamically?

Reply:
Yes, you have to create element as well from `javascript` and append to `html`.

Reply:
Okay, I'll do it. But there is no place on this part of the project saying to do that. https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md

Reply:
In the requirement it says `index.html` contain only root element.

![Screenshot from 2021-07-28 02-25-49](https://user-images.githubusercontent.com/22434660/127226185-98efe072-9d41-4e7a-a44a-93cc6b2726ff.png)

Reply:
Ok, maybe they should put that in the requirements then. 

Project requirements
- IMPORTANT NOTE This is the first time you will use Gitflow. Keep that in mind!
- Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).
- Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps.

- Don't make API calls (you'll do it in a following lesson).
- Use ES6 modules, with import and export.

Reply:
I'm doing it. I'm just complaining because this is not for us there.

Reply:
Okay, I will confirm with the team and thank you for that you are applying the changes. :slightly_smiling_face: